src/views/unauthenticated/NewUserHomeInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { SignInButton } from "../../api";
+
+export function NewUserHomeInfo() {
+	const navigate = useNavigate();
+
+	return (
+		<>
+			<h1>Welcome to Smart shopping list</h1>
+			<p>
+				The next best thing to having someone else do the shopping for you!
+				Create and manage smart lists learn your habits to let you know exactly
+				when you will need t
+			</p>
+
+			<article>
+				<p>New to Smart shopping list?</p>
+				<button
+					onClick={() => navigate("/about")}
+					aria-label="Navigate to the about application page."
+				>
+					Learn More
+				</button>
+			</article>
+
+			<article>
+				<p>Welcome Back:</p>
```

Comment:
This is both the sign-in and sign-up button, right? Maybe this wording could be changed to reflect that?

Reply:
I think a separate "sign up" button next to the learn more button  seems to fit  better as opposed to a single button for both sign in/sign ups? Adding the "sign up" button one more time in the learn more about page after the user has learned about the smart cart app, for extra nudging. Let me know what you think!

Reply:
I like this idea!